packages/testcontainers/src/generic-container/generic-container-logs.test.ts

```diff
@@ -26,6 +26,16 @@ describe("GenericContainer logs", () => {
     await container.stop();
   });
 
+  it("should stream logs with ContainerLogsOptions from a started container", async () => {
+    const container = await new GenericContainer("cristianrgreco/testcontainer:1.1.14").withExposedPorts(8080).start();
+    const currentYear = new Date().getFullYear().toString();
+
+    const stream = await container.logs({timestamps: true});
```

Comment:
Here you can see what I meant in my previous comment: if users pass
```
    const stream = await container.logs({timestamps: true, follow: false});
```

Will the false value be overridden by https://github.com/testcontainers/testcontainers-node/pull/670/files#diff-cde64c872b844fe1a7e8949d7eee46cea0cb346b40d840597c1a4785d68998d2R162?

Reply:
Yes, it will be overridden inside library and will be true

Reply:
Also it is already skipping opts passed to this method, because it's using only `since` parameter – [here](https://github.com/testcontainers/testcontainers-node/blob/main/packages/testcontainers/src/container-runtime/clients/container/docker-container-client.ts#L164)

It's not so easy as I thought firstly :)

@javierlopezdeancos I think easiest way will be to leave it as it now in [docker-container-client.ts](https://github.com/testcontainers/testcontainers-node/pull/670/files/992c3112c415b879cf151c29b2556478b9d8be1a#diff-cde64c872b844fe1a7e8949d7eee46cea0cb346b40d840597c1a4785d68998d2).  What do you think about that?

Reply:
To not breking changes I suggest to this

```javascript
const myOpts = {
  foo: "foo",
  bar: "bar"
}

const getOpts = (opts) => {
  return  {
    ...opts,
    foo: false
  }
}

console.log(getOpts(myOpts))

const getOptsOverrited = (opts) => {
  return  {
    ...opts,
    foo: opts?.foo ?? false
  }
}

console.log(getOptsOverrited(myOpts))
```

Review here the snippet https://jsbin.com/gofedomanu/edit?js,console

That in our code should be maintain the since pattern to each prop that we should set to the default state

```typescript
 try {
      log.debug(`Fetching container logs...`, { containerId: container.id });
      const stream = (await container.logs({
        ...opts,
        stdout: opts?.stdout ?? true,
        stderr: opts?.stderr ?? true,
        since: opts?.since ?? 0,
      })) as IncomingMessage;
      stream.socket.unref();
      const demuxedStream = this.demuxStream(container.id, stream);
      log.debug(`Fetched container logs`, { containerId: container.id });
      return demuxedStream;
    }
```

Reply:
reverted to previous version